packages/next/src/index.ts

```diff
@@ -1807,6 +1807,7 @@ export const build: BuildV2 = async ({
                 path.relative(baseDir, entryPath),
                 '___next_launcher.cjs'
               ),
+              operationType: 'SSR', // TODO: What should these be?
```

Comment:
These can be SSR, API or ISR. To infer ISR here would have to look at the `prerender-manifest`